pkg/controller/deadmanssnitch/deadmanssnitch_controller.go

```diff
@@ -171,10 +171,10 @@ func (r *ReconcileDeadMansSnitch) Reconcile(request reconcile.Request) (reconcil
 		for _, s := range snitches {
 			delStatus, err := r.dmsclient.Delete(s.Token)
 			if !delStatus || err != nil {
-				reqLogger.Info("Failed to delete the DMS from api.deadmanssnicth.com", "Namespace", request.Namespace, "Name", request.Name)
+				reqLogger.Info("Failed to delete the DMS from api.deadmanssnitch.com", "Namespace", request.Namespace, "Name", request.Name)
```

Comment:
"Namespace", request.Namespace, "Name", request.Name in the logging statement is redundant.

 reqLogger is created with values (reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)) in the start of reconcil loop and anything logged with reqLogger will always add the fields.

Reply:
Above comment applies to various other logging statements in the file as well